admin_guide/scheduling/priority_preemption.adoc

```diff
@@ -0,0 +1,153 @@
+[[admin-guide-priority-preemption]]
+= Pod Priority and Preemption
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+Pods can have priority. Priority indicates the importance of a Pod relative to other Pods. If a Pod cannot be scheduled, the scheduler tries to preempt (evict) lower priority Pods to make scheduling of the pending Pod possible.
+
+Priority also affects scheduling order of Pods and out-of-resource eviction ordering on the Node.Pod priority and preemption .
+
+== How to use priority and preemption
+
+To use priority and preemption, follow these steps:
+
+. Add one or more PriorityClasses.
+
+. Create Pods withpriorityClassName set to one of the added PriorityClasses. Of course you do not need to create the Pods directly; normally you would add priorityClassName to the Pod template of a collection object like a Deployment.
+
+
+If you try the feature and then decide to disable it, you must remove the PodPriority command-line flag or set it to false, and then restart the API server and scheduler. After the feature is disabled, the existing Pods keep their priority fields, but preemption is disabled, and priority fields are ignored. If the feature is disabled, you cannot set priorityClassName in new Pods
+
+== How to use priority and preemption
+
+Preemption is controlled by a scheduler flag disablePreemption, which is set to false by default.
+
+To disable preemption, set disablePreemption to true. This keeps pod priority enabled but disables preemption. Here is a sample configuration:
+
+.Sample component config with preemption disabled
```

Comment:
I think OpenShift still not support this usage for v3.11, so need update here.

Reply:
We don't have a resource type called component config in API server, these are configs loaded from disk. We can pass `kube-scheduler --config=/path/to/config` but I think this section is confusing in general as we do have openshift master config file from which we are loading the information.